xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 12.3.2
+JSONRPC_VERSION 12.3.3
```

Comment:
I don't see any change in this PR that needs a bump of the JSON-RPV API version. @DaveTBlake maybe?

Reply:
API bump is because the flag is available by requesting the streamdetails for a file. 
E.G.  A query of `{"jsonrpc": "2.0", "id": 1, "method": "VideoLibrary.GetMovieDetails","params":{"movieid":72,"properties":["streamdetails", "year"]}` will return a result now containing 
`"streamdetails": "video": [{"aspect": 2.4,"codec": "hevc", "duration": 7238, "hdrtype": "hdr10", "height": 1600, "language": "eng", "stereomode": "", "width": 3840}]`

Docs say "Bumped on any changes to the internal implementation but not to the API definition" so I think it's right, API hasn't changed but more data is serialized.

Reply:
Answering my own question: There is a JSON-RPC API change intended as you are exposing the new VideoInfoTag attribute "hdrtype" => https://github.com/xbmc/xbmc/pull/19983/files#diff-3c935d9995dddf9fb956a1cc48757d8bf79e06443afbb17bb0c15fb383eaf764R74

But you also need to update https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/json-rpc/schema/types.json . @DaveTBlake  can assist here for sure.

Reply:
> "Bumped on any changes to the internal implementation but not to the API definition"

I'd say, by returning "hdrtype" via JSON-RPC to any client, you changed the API definition, thus method schema must be adapted. But let's have @DaveTBlake the final saying on this.

Reply:
Yeah, I think you are right.  The field needs adding to` "Video.Streams": "video":{"properties"}` else the introspection will be wrong.  I _think_ that that is a bump to the minor version number.  So `12.4.0` but json stuff fries my brain at the best of times so I'm probably wrong!